src/auth/strategies/util.ts

```diff
@@ -81,7 +88,7 @@ export async function ExternalServiceCallback(
     }
 
     if (!user) {
-        done(null, false, { "message": "Could not match login to existing account" });
+        done(null, false, { message: "Could not match login to an existing account. Please try again with a different login method." });
```

Comment:
Honestly, if the third-party IdP email doesn't correspond to the email they entered originally, we should:

1. Just tell people that, or
2. Change the UX so you can directly pick 3rd-party IdPs on the initial login screen (which may have other implications I'm not thinking of right now)

Reply:
I don't know if picking 3rd party IdPs directly is an option anymore, since there are so many accounts whose 3rd party IdP email doesn't match their main email address on their account, so it wouldn't work.